tools/mochaRunner/src/interface.ts

```diff
@@ -28,8 +28,6 @@ const skippedTests: Array<{testIdPattern: string; skip: true}> = process.env[
   ? JSON.parse(process.env['PUPPETEER_SKIPPED_TEST_CONFIG'])
   : [];
 
-skippedTests.reverse();
```

Comment:
Why was this removed? the last matching pattern should apply, right?

Reply:
The TestExpectations are now reversed after we read them, that lets us simplify the search -> See [change].(https://github.com/puppeteer/puppeteer/pull/9870/files#diff-b7bd54671e35ed689bbd420e35dbd2b343b9595252279fa082f9e419a724c98bR109)